web/helpers.js

```diff
@@ -61,4 +62,11 @@ handlebars.registerHelper("alertFails", (dict, total) => {
     }
     return new handlebars.SafeString(output + "')");
 });
-// TODO: add helper for resourceLink 
\ No newline at end of file
+// for use in the list.hbs additional resource hyperlinks
+handlebars.registerHelper("resourceLinks", (str) => {
+    var links = "";
+    str.split('\n').forEach(element => {
+        links += "<a href=\"" + handlebars.escapeExpression(element) + "\">" + handlebars.escapeExpression(element) + "</a>";
```

Comment:
Single quotes are valid for use here instead of having to escape the doubles on the inside